### DISCUSSION TITLE
Enhancement - TPB vs regular series

### DISCUSSION THREAD

**Discussion by musictrivianut** [comictagger/comictagger#435] [Ideas]
Don't know if anyone has pointed this out already, but it seems that there are more than a few TPBs being published these days that have the same cover art as the very first issue of whatever series is being collected.  I have seen comictagger choke on which one a file is way too many times.

When using the gui, I can see that CT picks up on how many pages there are in a file (20-35 for a normal issue; hundreds sometimes for TPBs).  Would it be possible to use that as a check of some sort so it can tell which format a book should be?  I know that may get into weird territory with some of the director's cuts that are being issued, but I thought it might be worth asking, just in case.

Thanks much!!

**Comment by lordwelch** [comment]
There is actually already a similar check in, except it removes series with a single issue (TPBs) if the issue count is not 1.
Can you check PR #436 to see how well it works? You will need to run it with `--tpb-detection` eg `comictagger.exe --tpb-detection` to enable the feature
